test/e2e/specs/editor/various/pattern-overrides.spec.js

```diff
@@ -85,16 +85,17 @@ test.describe( 'Pattern Overrides', () => {
 				await advancedPanel.click();
 			}
 			await editorSettings
-				.getByRole( 'checkbox', { name: 'Allow instance overrides' } )
-				.setChecked( true );
+				.getByRole( 'textbox', { name: 'Block Name' } )
+				.click();
+			await page.keyboard.type( editableParagraphName );
```

Comment:
Nit: we can just do `.fill` here for simplicity.

```suggestion
				.fill( editableParagraphName );
```